packages/core/helper-plugin/lib/src/components/HeaderSearch/Wrapper.js

```diff
@@ -3,6 +3,7 @@ import styled from 'styled-components';
 const Wrapper = styled.div`
   position: fixed;
   top: 0;
+  left: 25rem;
```

Comment:
It seems that the UI of the Search component is different now than what it used to be.

Now:

![Screenshot 2021-06-15 at 11 00 40](https://user-images.githubusercontent.com/13311463/122024699-f60afa80-cdc8-11eb-8832-3ca736fa0dfc.png)

Before:

![Screenshot 2021-06-15 at 11 00 45](https://user-images.githubusercontent.com/13311463/122024718-facfae80-cdc8-11eb-8a35-876e722fde67.png)


Do you mind checking it?